ChatApp/src/screens/phone_auth/otp_signup/otp_verify.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useRef, useState, useEffect } from "react";
```

Comment:
Don't create a new component for the signup flow. Use the one used in sign-in flow. So, not reviewing the remaining changes in this PR.